index.js

```diff
@@ -89,10 +90,9 @@ function Pickup (opts) {
 
     if (isSet) {
       // First wins, except 'content:encoded' summary of reasonable length…
```

Comment:
`// First mappings key wins, except 'content:encoded' summary of reasonable length.`